rules/evasion/script-obfuscation.yara

```diff
@@ -113,9 +114,23 @@ rule php_obfuscated_concat : high {
     hash_2024_PHP_wlstncyj = "1a1c97594340ede77bc814670eaf35eaba861f1f9519038582416c704796da0a"
   strings:
     $php = /<\?[^x]/
```

Comment:
Would it make sense to update this to match the other `$php` strings (`$php = "<?php"`)?

Reply:
sounds good - I don't believe short tags are supported by default anywhere in 2024.